UnifiedApi/Worker/Grid/SessionPollingService.cs

```diff
@@ -44,6 +45,8 @@ namespace ArmoniK.DevelopmentKit.Worker.Grid;
 [MarkDownDoc]
 public class SessionPollingService
 {
+  private readonly Dictionary<string, string> tasksResultsMap_;
```

Comment:
Name not very explicit. Especially since we do not use many dictionnary and haven't defined a naming convention.
I propose `taskId2OutputId_`

Reply:
fixed